src/main/java/com/intuit/graphql/orchestrator/schema/ServiceMetadataImpl.java

```diff
@@ -96,6 +106,23 @@ public FederationMetadata getFederationServiceMetadata() {
     return federationMetadata;
   }
 
+  @Override
+  public boolean shouldUpdateOperationsOrFields() {
+    return shouldRemoveExternalFields() || containsRenamedFields();
```

Comment:
unless we need all three methods separately
shouldUpdateOperationsOrFields, shouldRemoveExternalFields & containsRenamedFields
we should only keep one method.

Reply:
If we need all 3, we can combine this into some kind of Option Pojo
DownstreamOptions or something else which will have these 3 booleans